docs/guide/snaps-development-guide.md

```diff
@@ -23,21 +23,29 @@ Executing `mm-snap --help` will provide detailed usage instructions.
 This guide assumes that you've completed the ["Getting Started" tutorial](./snaps.html#getting-started).
 :::
 
-So, you have installed [MetaMask Flask](https://metamask.io/flask), cloned the [@metamask/template-snap](https://github.com/MetaMask/template-snap) repository, and have served the "Hello, World!" snap locally.
+So, you have installed [MetaMask Flask](https://metamask.io/flask), cloned the [@metamask/template-snap-monorepo](https://github.com/MetaMask/template-snap-monorepo) repository, and have served the "Hello, World!" snap locally.
 It's time to develop your own snap.
 
 A snap is a JavaScript program that, conceptually, runs in a sandboxed environment inside MetaMask.
 At the moment, snaps must be distributed as npm packages on the official npm registry (`https://registry.npmjs.org/`), but different distribution mechanisms will be supported in the future.
 If you look at the directory structure of the template snap repository, you'll see that it looks something like this:
 
 ```text
-template-snap/
-├─ dist/
-│  ├─ bundle.js
+template-snap-monorepo/
+├─ packages/
+│  ├─ site/
+|  |  |- src/
+|  |  |  |- App.tsx
+|  |  ├─ package.json
+|  |  |- ...(react app content)
+|  |
+│  ├─ snap/
+|  |  ├─ src/
+|  |  |  |- index.ts
+|  |  ├─ snap.manifest.json
+|  |  ├─ package.json
+|  |  |- ... (snap content)
 ├─ package.json
-├─ snap.manifest.json
-├─ src/
-│  ├─ index.js
 ├─ ... (other stuff)
 ```
```

Comment:
Updated the file tree to reflect the `template-snap-monorepo` file tree

Reply:
@eriknson

Reply:
Looks good